package.json

```diff
@@ -159,7 +159,7 @@
     "settings": {
       "import/resolver": {
         "webpack": {
-          "config": "./internals/webpack/webpack.dev.babel.js"
+          "config": "./internals/webpack/webpack.prod.babel.js"
```

Comment:
My first thought was "this should be `webpack.base.babel.js`" as that is where the resolve is actually declared! But that does not work 😕 There seems to many issues open around this, do you know why it works with prod and no other webpack file @gihrig ?

I've tested before this change on Atom and VSCode on Windows and see the reported errors. Then applied your fix and can confirm it works on both 👍